chain/network/src/config.rs

```diff
@@ -96,6 +96,8 @@ pub struct NetworkConfig {
     pub whitelist_nodes: Vec<PeerInfo>,
     pub handshake_timeout: time::Duration,
 
```

Comment:
Make this comment a bit more implementation-agnostic, sth like: "Whether to re-establish reliable connections from the previous neard run. See `near_network::peer_manager::connection_store` for details."